backend/routes/XummApi.js

```diff
@@ -0,0 +1,424 @@
+import express from 'express'
+import xrpl from "xrpl";
+import fetch from 'node-fetch'
+import axios from 'axios'
+import {create} from 'ipfs-http-client';
+import XrplNFTHelper from './XrplNFTHelper.js';
+import XummSdk from 'xumm-sdk';
+
+
+const xummApi = ''
+const xummSecret = ''
+
+const pinataApiKey = ''
+const pinataSecretApiKey = ''

```

Comment:
These should probably move to a `.env` file, or read from environment variables to ensure people don't accidentally leave their passwords in a plaintext file.